### ISSUE TITLE
Difficulty with Free Worlds "Early Warning" mission

### ISSUE THREAD

**Issue by jerith** [endless-sky/endless-sky#8638]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

For my current game (continuous build), I decided to play through the Free Worlds campaign without a fleet or alien tech or whatever. All was going well until I reached the "Early Warning" mission that requires scouting several Navy-occupied systems. So far, all my attempts to complete it have ended in spectacular fiery death.

The problem is that entering a system, slowing to a stop, turning, and leaving for the next system often takes long enough that the giant fleet in the system has time to reach me before I can move on. If it were just one or two systems, reloading and trying again a few times would be sufficient to succeed. However, having to get lucky with entry point and fleet placement in six systems in a row (counting Wei twice, because it's the way home) with nowhere in between to land makes this an exercise in frustration.

I have no doubt I could manage it with a jump drive (which significantly reduces the time spent in each system) or with a bunch of escorts to help distract the Navy, or with an alien ship that can take more of a beating than anything available in the Free Worlds, but I don't see a way forward for a single ship without late-game tech.

(It's possible that I've missed some obvious way to complete this mission as-is, but if that's the case I expect other people will miss it too.)

### Steps to Reproduce

Play the Free Worlds Campaign in a single ship without alien or late-game tech until you reach the "Early Warning" mission. (Or use the attached save.)

### Expected Behavior

It should be possible to complete this mission without a major upgrade to your ship, or acquiring a fleet of escorts.

Ideally there would either be fewer systems to scout, somewhere safe to land between hostile systems, or smaller enemy fleets in the various systems.

### Link to save file

[Brae Sondet~3017-01-06-early-warning-difficulty.txt](https://github.com/endless-sky/endless-sky/files/11171909/Brae.Sondet.3017-01-06-early-warning-difficulty.txt)

### Operating System

MacOS

### Game Source

ESLauncher2

### Game Version

0915ad3

### Additional Information

I started the game a couple of days ago with whatever was in the continuous build then, but I don't see any relevant changes between then and now.

**Comment by jerith** [comment]
I was eventually able to complete this mission by stripping down my fastest ship to the bare essentials to maximize turn and acceleration, breaking up the route into a series of smaller runs instead of doing the whole circuit at once, and making a couple of dozen attempts until I got lucky.

While the mission isn't *impossible*, it's an unreasonable difficulty spike that will likely be a problem for new players.

**Comment by Hurleveur** [comment]
Were you using a scram drive? I feel like the fleets are pretty slow and if needed you can go away for a bit first?
What is your fastest ship? A light warship, a scout? Or smth heavier?

**Comment by jerith** [comment]
> Were you using a scram drive?

No, a hyperdrive. I've never found a scram drive to be worth the extra outfit space and fuel consumption. I'm not sure if it'll help here, but even if it does it's not an obvious solution to the problem.

> I feel like the fleets are pretty slow and if needed you can go away for a bit first?

In the time it takes to turn, slow down, and turn again, the Navy fleets are often within weapons range. Half the time I end up jumping in right next to a carrier or cruiser or something, and then no amount of speed can save me.

> What is your fastest ship? A light warship, a scout? Or smth heavier?

A splinter with atomic engines. I don't have the specs in front of me, but it's about the fastest medium warship I could build. I also tried with a Falcon in case it could survive long enough to leave the system despite being slower, but it blew up just as quickly. It might be possible to do it in a speed-optimized interceptor or something, but then rage would likely be an issue.

**Comment by Hecter94** [comment]
A scram drive would have been a big asset to you, as well as reverse thrusters. You can jump out as fast as a jump drive would allow with those two things. I believe there are various news hails in the game and other dialogue mentioning that scram drives and reverse thrusters help for fast jumps, so this will be the opportunity for a player to put that into practice.

**Comment by Hurleveur** [comment]
Yeah a scram drive changes everything - you said it yourself that your ship needs to slow down
But we could also make the navy fleets a bit more timid?

**Comment by jerith** [comment]
I just tried a scram drive (without reversers) and it makes the mission *much* easier -- first time success. If the intention is for the player to use a scram drive for this, it should probably be mentioned somewhere in the mission text or introductory conversation.

On a related note, aside for very specific scenarios (like this one) the additional cost of a scram drive in both fuel consumption and outfit space is way too high. I'll open a separate issue for that.

**Comment by jerith** [comment]
Installing a scram drive for this mission is rather annoying. Deep doesn't have an outfitter, so we're going to have to go somewhere else to get it. Three neighbouring systems systems have planets with outfitters:

* Trinket stocks scram drives, but not fuel pods.
* Glaze stocks fuel pods, but not scram drives.
* Longjump stocks neither scram drives nor fuel pods.

In order to reach Alphecca and return to a safe system for refueling (or ramscooping), you need to be able to make six jumps. With a scram drive, this requires 900 fuel storage. (800 and a single ramscoop isn't enough, and ramscoops are both bigger than fuel pods and not available in the Free Worlds anyway.) For an average ship that's not already configured for extremely long hyperdrive range, this means visiting outfitters in two different systems or making the six hop round trip to Zug (which is the closest shipyard with both).

Since this is the only mission (or one of very few, at least) that requires a scram drive, being able to buy both the drive and the necessary fuel pods in the same place would be nice.

The only (human) outfitter definition with fuel pods is "Common Outfits", while scram drives are available in "Pirate Outfits" and four Advanced/Security variants. Unless there's a specific reason not to have pirates sell fuel pods, I think adding them to "Pirate Outfits" (or maybe even "Basic Outfits") would make sense both for this mission and in general.
